ls1intum/Artemis
# More Feedback Request

### Checklist

### Motivation and Context
It allows user to request more feedback for text and modeling exercises.

### Description
Student can only open up to 3 complaints, but sometimes student wants only to get more feedback on his score, that is where this feature is useful.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to any course with submitted Text Exercise
3. Open Text Exercise and click Request More Feedback
4. Enter the reason for request and click 'Request More Feedback' (1st screenshot)
5. Navigate to the same exercise in Tutor Dashboard
6. Click 'Open Assesment' and you can see More Feedback request on the bottom (2nd screenshot)
7. After more feedback was provided, you will see that request is accepted (3rd screenshot)
8. Navigate back to submitted exercise and you can see respond to your request (4th screenshot)

### Screenshots
![1st screenshot](https://user-images.githubusercontent.com/17262376/59967744-0f0ca800-952f-11e9-8774-791d6e9706eb.png)
---
![2nd screenshot](https://user-images.githubusercontent.com/17262376/59967766-2a77b300-952f-11e9-8769-e7d3ee464bc3.png)
---
![3rd screenshot](https://user-images.githubusercontent.com/17262376/59967769-3b282900-952f-11e9-9741-cdd14ada2f36.png)
---
![4th screenshot](https://user-images.githubusercontent.com/17262376/59967771-3fecdd00-952f-11e9-8f92-fbbd522f9fbb.png)
---
#### Tutor Exercise Dashboard
![Screenshot_2019-07-11 Tutor Exercise Dashboard](https://user-images.githubusercontent.com/17262376/61080440-5ba81c80-a425-11e9-8301-66ed45d3326b.png)
---
#### Tutor Course Dashboard
![Screenshot_2019-07-20 Tutor Course Dashboard](https://user-images.githubusercontent.com/17262376/61581498-41ee8f80-ab1f-11e9-9a74-e3a9b47db857.png)

---
#### List of More Feedback Requests
![Screenshot_2019-07-11 List of more feedback requests](https://user-images.githubusercontent.com/17262376/61083649-995c7380-a42c-11e9-8c4e-aacbf1266cb1.png)
---
#### Instructor Exercise Dashboard
![Screenshot_2019-07-11 Instructor Exercise Dashboard](https://user-images.githubusercontent.com/17262376/61084238-ce1cfa80-a42d-11e9-9a59-904776e8d636.png)
---
#### Instructor Course Dashboard
![Screenshot_2019-07-12 Instructor Course Dashboard](https://user-images.githubusercontent.com/17262376/61089173-99fc0680-a43a-11e9-961b-39281f7a8b25.png)
